man/tpm2_changepps.1.md

```diff
@@ -0,0 +1,42 @@
+% tpm2_changepps(1) tpm2-tools | General Commands Manual
+
+# NAME
+
+**tpm2_changepps**(1) - Replaces the active platform primary seed with a new one
+generated off the TPM2 RNG.
+
+# SYNOPSIS
+
+**tpm2_changepps** [*ARGUMENT*]
+
+# DESCRIPTION
+
+**tpm2_changepps**(1) - Replaces the active platform primary seed with a new one
+generated off the TPM2 RNG. The Transient and Persistent objects under the
+platform hierarchy are lost whilst retaining the NV objects.
+
+# OPTIONS
+
+  * **ARGUMENT** the command line argument specifies the _AUTH_ for the platform
+  hierarchy.
+
+## References
+
+[authorization formatting](common/authorizations.md) details the methods for
+specifying _AUTH_.
+
+[common tcti options](common/tcti.md) collection of options used to configure
+the various known TCTI modules.
+
+# EXAMPLES
+
+## Change the platform primary seed where in the platform auth is NULL.
```

Comment:
Change the platform primary seed where the platform authorization value is empty.